recipes/doctest/2.x.x/conanfile.py

```diff
@@ -20,6 +21,9 @@ def source(self):
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    def build(self):
```

Comment:
I don't care about that warning, because it's a header-only recipe and doesn't require a build step. But at same time, adding it doesn't harm.